UnityProject/Assets/Light2D/Editor/Light2DMenu.cs

```diff
@@ -75,7 +75,7 @@ public static void Enable2DToolkitSupport()
         [MenuItem("GameObject/Light2D/Disable 2DTK Support", false, 6)]
         public static void Disable2DToolkitSupport()
         {
-            BuildTargetGroup[] targets = (BuildTargetGroup[]) Enum.GetValues(typeof(BuildTargetGroup));
+            var targets = (BuildTargetGroup[]) Enum.GetValues(typeof(BuildTargetGroup));
```

Comment:
This cleans up the codebase a lot

Reply:
Yeah, shortening these, as well as Lists and Dictionaries is fine

Reply:
Will add it atleast to the codingstyle this way, even if we don't use the automated cleanup.